concepts/index.md

```diff
@@ -69,3 +107,20 @@ Spinnaker treats cloud-native deployment strategies as first class constructs, h
 
 > Learn more about deployment management on the [Pipelines](/concepts/pipelines/) page.
 
+## The application-stack-detail naming convention
+
+Applications, clusters, and server groups following the
+application-stack-detail-version naming convention.
+
+Resources you deploy using Spinnaker follow this convention naturally, because
+you assign these names (not including version) when you create those resources.
+If you try to use infrastructure, you will have to rename some resources to use
```

Comment:
What do you mean by "if you try to use infrastructure" here? If you try to use infrastructure that you've created outside of Spinnaker?

Reply:
Fixed. Typo. I meant if you try to use pre-existing infrastructure.